components/FA2Transfer.tsx

```diff
@@ -96,8 +96,7 @@ const FA2Transfer = ({
   const fetchTokens = useCallback(
     (value: string, offset: number) =>
       fetch(
-        // `${API_URL}/v1/tokens/balances?account=${state.currentContract}&offset=${offset}&limit=20&token.metadata.name.as=*${value}*`
-        `${API_URL}/v1/tokens/balances?account=tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb&offset=${offset}&limit=20&token.metadata.name.as=*${value}*`
+        `${API_URL}/v1/tokens/balances?account=${state.currentContract}&offset=${offset}&limit=${FETCH_COUNT}&token.metadata.name.as=*${value}*&balance.ne=0&sort.desc=id`
```

Comment:
I am thinking maybe `&sort.desc=lastTime`make more sense  than `&sort.desc=id` ?  WDYT?

Reply:
sorry, and one more parameter should be add `&standard=fa2`

Reply:
update: `&token.standard.eq=fa2`